loopy/schedule/__init__.py

```diff
@@ -571,6 +571,7 @@ class SchedulerState(ImmutableRecord):
 
     .. attribute:: loop_priority
 
+        #FIXME: incorrect docs.
```

Comment:
In [!388](https://gitlab.tiker.net/inducer/loopy/-/merge_requests/388) @inducer said:
> https://docutils.readthedocs.io/en/sphinx-docs/ref/rst/restructuredtext.html#comments

Reply:
Fixed via 9b0a54c. (loop_priority wasn't an attribute of SchedulerState)